Packs/Reco/ReleaseNotes/1_1_0.md

```diff
@@ -5,4 +5,4 @@
 
 - Get new type of alerts
 - Get magic link to Reco portal
-- Updated the Docker image to: *demisto/python3:3.10.11.58677*.
+- Updated the Docker image to: demisto/python3:3.10.11.59581
```

Comment:
please revert this change :)
you should have only one releaseNote file for each pr